src/System.Security.Cryptography.Pkcs/src/Resources/Strings.resx

```diff
@@ -100,4 +100,7 @@
   <data name="InvalidOperation_WrongOidInAsnCollection" xml:space="preserve">
     <value>AsnEncodedData element in the collection has wrong Oid value: expected = '{0}', actual = '{1}'.</value>
   </data>
+  <data name="PlatformNotSupported_CryptographyPkcs" xml:space="preserve">
+    <value>System.Security.Cryptography.Pkcs is not yet supported on this platform</value>
```

Comment:
Although the developer is using an API in the `Pkcs` namespace it might be helpful to expand it, eg.,
`Public Key Cryptography Standards (PKCS) APIs are not supported on this platform.`.

Please avoid "not yet" in messages - we don't want to mislead into suggesting we may support a platform in future when we may not.

Please consistently end messages with periods, or not.